BUILD.bazel

```diff
@@ -183,6 +183,42 @@ cc_library(
 
 # === End of rpc definitions ===
 
+cc_library(
+    name = "ray_common",
+    srcs = glob(
+        [
+            "src/ray/common/**/*.cc",
+        ],
+        exclude = [
+            "src/ray/common/**/*_test.cc",
+        ],
+    ),
+    hdrs = glob(
+        [
+            "src/ray/common/**/*.h",
+        ],
+    ),
+    copts = COPTS,
+    deps = [
+        ":common_cc_proto",
+        ":node_manager_fbs",
```

Comment:
will this be removed later?

Reply:
also add a comment on the core_worker section that the dependency to raylet lib will be removed later?

Reply:
yes, it'll be removed.